dist/clouduploader.rb

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/ruby
+require 'json'
+require 'ostruct'
+require 'open3'
+
+if ARGV.length != 5
+  raise 'Wrong number of arguments, please provide: user platform upload_file targetdata filename'
+end
+
+platform = ARGV[1]
+image_path = ARGV[2]
+data_path = ARGV[3]
+filename = ARGV[4]
+
+data = JSON.parse(File.read(data_path))
+DURATION = 60 * 60
```

Comment:
This will always be the same right? :-)

Reply:
It is the maximum of 1 hour for now. Maybe we can decrease it in the future when we know how long an upload usually takes.

Reply:
Thinking about it... DURATION_IN_SECONDS might be easier to understand.

And Henne means that you could just write 3600 ;-)